Clarify that `mod-id` placeholders must be customized (#348)
FabricMC/fabric-docs: develop/getting-started/creating-a-project.md

The package name should be lowercase, separated by dots, and unique to avoid conflicts with other programmers' packages. It is typically formatted as a reversed internet domain, such as `com.example.mod-id`.

::: warning
Make sure you remember your mod's ID!

> Whenever you find `mod-id` in these docs, especially in file paths, you will have to replace it with your own. For example, if your mod's id is 'my-first-mod', and some path shoud be `resources/assets/my-first-mod`, not `resources/assets/mod-id`. 

Reviewer: ```suggestion
Whenever you find `mod-id` in these docs, especially in file paths, you will have to replace it with your own.

For example, if your mod ID was **`fabric-docs-reference`**, instead of _`resources/assets/mod-id`_ use **`resources/assets/fabric-docs-reference`**.
```

Author: redundancy just wnat to make more clear

:::

> 

Reviewer: ```suggestion
```
